### ISSUE TITLE
"PhyFS failed to initialise: (null)" error when starting SAIS 1.6.2 in Windows 10

### ISSUE THREAD

**Issue by penyuan** [kuroneko/sais#6]
Many thanks to the developers for releasing SAIS under the GPL and for the latest [1.6.2](https://github.com/kuroneko/sais/releases/tag/v1.6.2) release!

Before 1.6.2, starting SAIS on my Windows 10 system fails and exits silently without any feedback. Now it still fails to start but gives a "PhyFS failed to initialise: (null)" error.

What does this mean? How can I help troubleshoot and solve this problem? Thank you!

**Comment by kuroneko** [comment]
Can you confirm how you're installing SAIS GPL and how you launched it?  If you're installing from the .zip file, can you please capture a screenshot of an explorer window in the SAIS folder in the detail file list view so I can see how it's unpacked?

That failure message is only possible right at the very start of launching - before anything has initialised - and is probably related to an installation problem, but I need some more detail first before I know for certain.

**Comment by kuroneko** [comment]
Actually - I've found there's a bug in physfs 3.0.1 that may be responsible for this.   I've requested upstream (bincrafters) to update to 3.0.2 - I'll see about producing an interrim build with an updated physfs.

**Comment by penyuan** [comment]
> Can you confirm how you're installing SAIS GPL and how you launched it? If you're installing from the .zip file, can you please capture a screenshot of an explorer window in the SAIS folder in the detail file list view so I can see how it's unpacked?

Thanks for getting back to me so quickly!

I saw your comment about physfs 3.0.1, but in case it still helps here's my screenshot of the 1.6.2 .zip file unpacked in Windows 10:

![SAIS-GPL-1.6.2-Windows-AMD64.zip unpacked in Windows 10](https://pic.infini.fr/49zUK3sC/Ga7PSPW0)

To be clear, the file I downloaded was [`SAIS-GPL-1.6.2-Windows-AMD64.zip`](https://github.com/kuroneko/sais/releases/download/v1.6.2/SAIS-GPL-1.6.2-Windows-AMD64.zip) directly from the [1.6.2 release page](https://github.com/kuroneko/sais/releases/tag/v1.6.2).

P.S. I tried downloading the 1.6.1 release (.zip and .msi) from Itch.io and it's got the problem of failing to start *without* showing an error message. Not sure if this is related...
